api/src/main/java/uk/gov/dhsc/htbhf/eligibility/factory/EligibilityResponseFactory.java

```diff
@@ -49,10 +54,27 @@ public EligibilityResponse createEligibilityResponse(DWPEligibilityResponse dwpR
 
         builder.numberOfChildrenUnderOne(nullSafeMax(dwpResponse.getNumberOfChildrenUnderOne(), hmrcResponse.getNumberOfChildrenUnderOne()));
         builder.numberOfChildrenUnderFour(nullSafeMax(dwpResponse.getNumberOfChildrenUnderFour(), hmrcResponse.getNumberOfChildrenUnderFour()));
+        builder.children(createChildren(dwpResponse, hmrcResponse));
 
         return builder.build();
     }
 
+    private List<ChildDTO> createChildren(DWPEligibilityResponse dwpResponse, HMRCEligibilityResponse hmrcResponse) {
+        List<ChildDTO> dwpChildren = getListOfEmptyIfNull(dwpResponse.getChildren());
+        List<ChildDTO> hmrcChildren = getListOfEmptyIfNull(hmrcResponse.getChildren());
+
+        if (dwpChildren.size() >= hmrcChildren.size()) {
+            return dwpResponse.getChildren();
+        }
+
+        return hmrcResponse.getChildren();
+    }
+
+    // Return the list if it's not null, otherwise return any empty list.
+    private List<ChildDTO> getListOfEmptyIfNull(List<ChildDTO> children) {
+        return firstNonNull(children, ImmutableList.of());
+    }
+
     private Integer nullSafeMax(Integer count1, Integer count2) {
```

Comment:
As discussed with David, there's a reason we're taking the maximum number of children from either response. When someone has a child (or adopts etc) they will move from CTC to UC, so in the cycle in which they transfer the CTC information will probably be out of date (having one fewer children) than UTC.

Reply:
Yes, that's fine, is this documented somewhere? If not, can we make sure that it is?

Reply:
Updating the top level javadoc explaining the rules regarding children.